server/neptune/gateway/event/check_run_handler.go

```diff
@@ -62,15 +62,19 @@ func (h *CheckRunHandler) Handle(ctx context.Context, event CheckRun) error {
 		return fmt.Errorf("event action type does not match string type.  This is likely a code bug")
 	}
 
-	status, err := toPlanReviewStatus(action)
+	if action.Identifier == "Unlock" {
```

Comment:
toPlanReviewStatus is already a function that exists to check this identifier. It feels like you can easily move things around and rename things to keep this in one place.  Why are you hijacking it here instead?

Reply:
moved it to one place